src/components/RouteLayout.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Switch from 'react-router-dom/Switch';
+import Route from 'react-router-dom/Route';
+
+const RouteWithSubRoutes = route => {
+	if (!route.component) {
+		throw new Error(`route for ${route.path} must have a 'component' property`);
+	}
+	if (route.render || route.children) {
+		console.warn('route.render and route.children function not supported');
+	}
+	return (
+		<Route
+			path={route.path}
```

Comment:
I redid my `yarn link` and updated some package json version numbers on this branch, and the error has gone away. I think it was just pointing to some wrong version of a module somewhere.